rules/integrations/github/impact_github_repository_deleted.toml

```diff
@@ -24,7 +24,12 @@ name = "GitHub Repository Deleted"
 risk_score = 47
 rule_id = "345889c4-23a8-4bc0-b7ca-756bd17ce83b"
 severity = "medium"
-tags = ["Domain: Cloud", "Use Case: Threat Detection", "Tactic: Impact", "Data Source: Github"]
+tags = ["Domain: Cloud", 
```

Comment:
This will bump rule versions. Anything else we can possibly add as part of the update and version consumption?

Reply:
Not that I can think of, I can revert the changes

Reply:
Oh actually the change here is adding the UEBA tag which is necessary for the threshold rule